amxmodx/amxmodx.cpp

```diff
@@ -194,10 +194,10 @@ static cell AMX_NATIVE_CALL console_print(AMX *amx, cell *params) /* 2 param */
 
 	int len;
 	char* message = format_amxstring(amx, params, 2, len);
-
+	
 	if (index < 1 || index > gpGlobals->maxClients)	// Server console
 	{
-		if (len > 254)
+		if (len > 254) // Server console truncates after byte 255. (254 + \n = 255)
```

Comment:
I just tried `EngFunc_ServerPrint` and it doesn't truncate after byte 255.

Reply:
I just paste the old comment. Just tested fastly and the limit should be 1024. Will make another PR for that, it's untouched and unrelated to new lines.